artifactory/commands/generic/upload.go

```diff
@@ -126,14 +160,23 @@ func (uc *UploadCommand) upload() error {
 		return err
 	}
 
-	// Build Info
-	if isCollectBuildInfo && !uc.DryRun() {
-		buildArtifacts := convertFileInfoToBuildArtifacts(filesInfo)
-		populateFunc := func(partial *buildinfo.Partial) {
-			partial.Artifacts = buildArtifacts
-			partial.ModuleId = uc.buildConfiguration.Module
+	if !uc.DryRun() {
+		// handle sync-delete
+		if uc.SyncDeletePath() != "" {
+			err = uc.handleSyncDelete(syncDeleteProp)
+			if err != nil {
+				log.Error(err)
```

Comment:
The error here is being "swallowed" instead of returned.

Reply:
I wanted to find a way to let build info part to run in case of am error...
Should I let it run antway or return an error?